operating/manage-user-access/configure-ocp-oauth.adoc

```diff
@@ -0,0 +1,27 @@
+[id="configure-ocp-oauth"]
+= Configuring {ocp} OAuth server as an identity provider
+include::modules/common-attributes.adoc[]
+:context: configure-ocp-oauth
+
+toc::[]
+
+[role="_abstract"]
+{ocp} includes a built-in OAuth server and you can use it as an authentication provider for {product-title} (RHACS).
```

Comment:
This is only applicable when you install using roxctl on OpenShift. This is never applicable if you are using helm or the operator.  The only time you would ever follow the procedure is when you use roxctl on OpenShift to install (which is highly unlikely.) This should go in the upgrade guide and apply only to OpenShift. I added the comments to the upgrade PR.